1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_Fizzbuzz/src/domain/utils/fizzbuzz.ts

```diff
@@ -0,0 +1,25 @@
+import { ArrayHasInvalidNumbersException } from "../exceptions/array-has-invalid-numbers.exception";
+
+export function fizzbuzzReplacement (sequence: number[]): string {
```

Comment:
should take one input that is a number (based on the README spec)

Reply:
Thanks, switched to receive a single number in v2